dimod/bqm/shapeablebqm.pyx.src

```diff
@@ -914,6 +914,52 @@ cdef class cyAdj@name@BQM:
 
         self.bqm_.set_linear(vi, b)
 
+    def add_constraint_as_quadratic(self, object terms, Bias
```

Comment:
Methods should be in alphabetical order (after special methods and properties). Private methods can be placed near their main methods.